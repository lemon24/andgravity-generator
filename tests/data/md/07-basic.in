*em* **strong**
[a](href)
![img](src)

* ul

1. ol
