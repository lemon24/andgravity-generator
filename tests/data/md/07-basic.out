<p><em>em</em> <strong>strong</strong>
<a href="href">a</a>
<img src="src" alt="img" /></p>
<ul>
<li>ul</li>
</ul>
<ol>
<li>ol</li>
</ol>
